Podfile

```diff
@@ -28,7 +28,8 @@ def wordpress_authenticator_pods
   ##
   pod 'Gridicons', '~> 1.0-beta' # Don't change this until we hit 2.0 in Gridicons
   pod 'WordPressUI', '~> 1.7-beta' # Don't change this until we hit 2.0 in WordPressUI
-  pod 'WordPressKit', '~> 6.0-beta' # Don't change this until we hit 5.0 in WPKit
+  # pod 'WordPressKit', '~> 7.0-beta' # Don't change this until we hit 8.0 in WPKit
+  pod 'WordPressKit', git: 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', branch: 'add/media-videopress-token'
```

Comment:
⚠️ `WordPressKit` pod will point to `~> 7.0-beta` once https://github.com/wordpress-mobile/WordPressKit-iOS/pull/581 is merged.

Reply:
Updated in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/754/commits/7eebe085b7a544e7091037c39974d606076dfe6d.